doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1446,18 +1454,16 @@ tactic language. Tactic notations obey the following syntax:
 
 .. cmdv:: Local Tactic Notation
 
-   Tactic notations do not survive the end of sections. They survive
-   modules unless the command Local Tactic Notation is used instead of
-   Tactic Notation.
+   Tactic notations disappear when a section is closed. They survive when
+   a module is closed unless the command ``Local Tactic Notation`` is used instead
+   of :cmd:`Tactic Notation`.
 
 .. rubric:: Footnotes
 
 .. [#and_or_levels] which are the levels effectively chosen in the current
    implementation of Coq
 
-.. [#no_associativity] Coq accepts notations declared as ``no
-   associativity`` but the parser on which Coq is built, namely
-   Camlp5, currently does not implement ``no associativity`` and
-   replaces it with ``left associativity``; hence it is the same for
-   Coq: ``no associativity`` is in fact ``left associativity``, for
-   the purposes of parsing
+.. [#no_associativity] Coq accepts notations declared as non-associative but the parser on
+   which Coq is built, namely Camlp4, currently does not implement ``no associativity`` and
+   replaces it by a ``left associativity``; hence it is the same for Coq: ``no associativity``
+   is in fact ``left associativity``.
```

Comment:
There are still problems with the conflict resolution here. Camlp4 had been changed in Camlp5 and this has been reverted, the part "for the purposes of parsing" had been added and this has been removed. A "a" had been removed and has reappeared. Given the recent changes to this paragraph, you can just keep it unchanged in this PR.